### DISCUSSION TITLE
console error when try load excalidraw

### DISCUSSION THREAD

**Discussion by Sohailamjad1** [excalidraw/excalidraw#5060] [Q&A]
react-dom.development.js:19662 The above error occurred in the <App> component:
    in App
    in div

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries.

**Comment by Sohailamjad1** [comment]
please help me to resolve this issue..
just because of this error ui is not rendering

**Comment by Sohailamjad1** [comment]
Hi @ad1992  after using production ..its working fine..earlier i was using development so some time its breaking...can you please suggest me which one i have to use????
<script
  type="text/javascript"
  src="https://unpkg.com/@excalidraw/excalidraw/dist/excalidraw.production.min.js"
></script>

**Comment by ad1992** [comment]
Can you try debugging the error in dev mode? I think its mostly related to your app's code and not Excalidraw, in prod also the error must be happening but it's not breaking the build (can you check the console)? If you are using directly in browser then you should use prod build and for debugging dev build, how ever if you are using the npm package then it will load dev build in dev and prod build in prod by itself.

**Reply by Sohailamjad1** [reply]
yea i tried with debugging and this is the console error iam getting 
**react-dom.development.js:19662 The above error occurred in the component:
in App
in div

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://fb.me/react-error-boundaries to learn more about error boundaries.**..
in dev mode ..and this issue iam not facing everytime...some time this issue is coming and some time not...wht should be the proper solution i don't know...
